python/cuml/test/test_linear_model.py

```diff
@@ -234,3 +234,42 @@ def test_logistic_regression_model_default(dtype):
     sklog.fit(X_train, y_train)
 
     assert culog.score(X_test, y_test) >= sklog.score(X_test, y_test) - 0.022
+
+
+@pytest.mark.parametrize('dtype', [np.float32, np.float64])
+@pytest.mark.parametrize('nrows', [10, 100])
+@pytest.mark.parametrize('column_info', [(20, 10)])
+@pytest.mark.parametrize('num_classes', [2, 10])
+@pytest.mark.parametrize('fit_intercept', [True, False])
+def test_logistic_regression_decision_proba(dtype, nrows, column_info,
```

Comment:
It would be easier to maintain if we broke these into two separate tests by feature- `test_predict_proba` (and `predict_log_proba`) and `test_decision_function`. That way more asserts can be added to them later as needed, based on the feature being tested.